lib/dispatcher/client.js

```diff
@@ -1475,10 +1399,15 @@ function shouldSendContentLength (method) {
 
 function write (client, request) {
   if (client[kHTTPConnVersion] === 'h2') {
-    writeH2(client, client[kHTTP2Session], request)
-    return
+    // TODO (fix): Why does this not return the value
+    // from writeH2.
+    writeH2(client, request)
```

Comment:
@metcoder95 This is confusing to me. Seems to have been broken for a while.

Reply:
Yeah, it seems its kind of flaky there; once wrap with the interceptors I'll jump right away into H2